### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by MaxHillebrand**
I think I went through all docs, and caught most typos there.

I did not consider typos fixed in #145, I hope there won't be any conflicts.

### CODE REVIEW

File: docs/developer/experimental-repository.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ I will not hand-over the instructions to random newbies though!
 
 ## Consequences of using packages in experimental
 
-* Risk of downtimes, including long (bitcoin chain resync)
+* Risk of downtime, including long (bitcoin chain resync)
```
Comment (Kixunil) [inline docs/developer/experimental-repository.md:16]: There could be multiple downtimes. Not sure what's best stylistically.

File: docs/security/faq.md [hunk 2]
```diff
@@ -83,7 +83,7 @@ No service can access data of other services except when it had to be allowed -
 Groups are also used for controlling access.
 This is very flexible as you can give any user/service access to more things by adding them to appropriate group.
 
-## Aren't users unsecure? What are the limitation of this design?
+## Aren't users insecure? What are the limitation of this design?
```
Comment (Kixunil) [inline docs/security/faq.md:86]: "unsecure" is correct term here as in non-secured. "insecure" is property of a person who is not confident about themselves.

File: docs/security/faq.md [hunk 3]
```diff
@@ -95,7 +95,7 @@ Such is the case of recent `sudo` vulnerability.
 
 This project mitigates these issues by:
 
-* Automating creation of filesystem items and setting proper permissions by default. This removes the possibility of mistakes.
+* Automating creation of file-system items and setting proper permissions by default. This removes the possibility of mistakes.
```
Comment (Kixunil) [inline docs/security/faq.md:98]: "filesystem" is term usually used by programmers. using hyphen doesn't feel right. I believe I've seen it with space somewhere.

**Review by Kixunil — CHANGES_REQUESTED** [review]
I believe at least "unsecure" should be resolved - see comments.

File: docs/security/faq.md (hunk 2 above)
Comment (MaxHillebrand) [reply docs/security/faq.md:86]:
```suggestion
## Aren't users unsecure? What are the limitation of this design?
```

File: docs/security/faq.md (hunk 3 above)
Comment (MaxHillebrand) [reply docs/security/faq.md:98]:
```suggestion
* Automating creation of filesystem items and setting proper permissions by default. This removes the possibility of mistakes.
```

File: docs/developer/experimental-repository.md (hunk 1 above)
Comment (MaxHillebrand) [reply docs/developer/experimental-repository.md:16]: the plural form of `downtime` is also `downtime`.
https://www.wordhippo.com/what-is/the-plural-of/downtime.html

Comment (Kixunil) [reply docs/developer/experimental-repository.md:16]: Ah, TIL, OK.

**Review by Kixunil — APPROVED** [review]
Thanks!

**Pull request merged**